Libraries/LLM/Models.swift

```diff
@@ -130,7 +130,8 @@ extension ModelConfiguration {
 
     public static let mistral7B4bit = ModelConfiguration(
         id: "mlx-community/Mistral-7B-Instruct-v0.3-4bit",
-        defaultPrompt: "Describe the Swift language."
+        defaultPrompt: "Describe the Swift language.",
+        extraEOSTokens: ["<|im_end|>"]  // !! This doesn't seem to work
```

Comment:
This doesn't remove it from the output -- it tells it to stop producing tokens if it is seen.

Reply:
I just don't understand why this token is now appearing at the end of the output when the chat template is used, which wasn't the case before. Are applications expected to remove it? I remember another model (maybe Gemma 2) that had a similar issue with the output, but I can't find it at the moment.